tree/ntuple/v7/inc/ROOT/RNTupleReader.hxx

```diff
@@ -272,12 +274,19 @@ public:
    template <typename T>
    RNTupleView<T> GetView(std::string_view fieldName)
    {
-      auto fieldId = fSource->GetSharedDescriptorGuard()->FindFieldId(fieldName);
-      if (fieldId == kInvalidDescriptorId) {
-         throw RException(R__FAIL("no field named '" + std::string(fieldName) + "' in RNTuple '" +
-                                  fSource->GetSharedDescriptorGuard()->GetName() + "'"));
-      }
-      return GetView<T>(fieldId);
+      return GetView<T>(RetrieveFieldId(fieldName));
+   }
+
+   template <typename T>
+   RNTupleView<T> GetView(std::string_view fieldName, std::shared_ptr<T> objPtr)
+   {
+      return GetView<T>(RetrieveFieldId(fieldName), objPtr);
+   }
+
+   template <typename T>
+   RNTupleView<T> GetView(std::string_view fieldName, T *rawPtr)
+   {
+      return GetView<T>(RetrieveFieldId(fieldName), rawPtr);
    }
```

Comment:
We can consider removing this overload. Users could still call `GetView<T>("fieldname", nullptr)` and then use `BindRawPtr` on the returned view. It would make the raw pointer interface slightly less prominent.